.devcontainer/test.sh

```diff
@@ -22,8 +22,11 @@ fi
 
 if [[ "$1" == intro/http-client ]]; then
     $HOME/.cargo/bin/cargo build --example http_client
-    # TODO: Update sdkconfig.defaults before buidling
-    # $HOME/.cargo/bin/cargo build --example https_client
+    # HTTPS Client example requires updating sdkconfig.defaults before buidling
+    sed -i 's/CONFIG_MBEDTLS_CERTIFICATE_BUNDLE=n/CONFIG_MBEDTLS_CERTIFICATE_BUNDLE=y/g' sdkconfig.defaults
```

Comment:
Is there a downside to having these set to `Y` by default?

Reply:
I think `CONFIG_MBEDTLS_CERTIFICATE_BUNDLE=n` came from some bug around the time of idf v4.4, when a `cryptography` python package update happened and the certificate bundle build stopped working due to an invalid certificate.
We can remove this config now and keep the certificate bundle enabled by default, same as it is for IDF projects.